iquise/urls.py

```diff
@@ -34,6 +34,7 @@
         name="favicon"
     ),
     url(r'^admin/', admin.site.urls),
+    url(r'^accounts/', include('django.contrib.auth.urls')),
```

Comment:
i get redirected to this url but not to a new page when i click on "forgot password"